scss/_variables.scss

```diff
@@ -7,11 +7,11 @@ $base-font-family: $serif;
 $crimson: #631010;
 $orange: #f77b42;
 
-$federal-blue: #022c53;
+$federal-blue: #112e51; /* U.S. Web Design Standards color primary-darkest */
```

Comment:
This is a good catch. @jenniferthibault can you change comments throughout to be: `// Comment` rather than: `/* Comment */` ? These comments will be ignored in the compiled CSS, which keeps the file size smaller overall.